--- a/includes/common.inc	2013-04-03 23:29:52.000000000 +0200
+++ b/includes/common.inc	2013-08-06 14:52:33.851153925 +0200
@@ -487,7 +487,7 @@
   $params = array();
 
   foreach ($query as $key => $value) {
-    $key = ($parent ? $parent . '[' . rawurlencode($key) . ']' : rawurlencode($key));
+    $key = ($parent ? $parent .  rawurlencode('[' .$key.']')  : rawurlencode($key));
 
     // Recurse into children.
     if (is_array($value)) {
