diff --git a/l10n_client.js b/l10n_client.js
index def53e7..a78afc4 100644
--- a/l10n_client.js
+++ b/l10n_client.js
@@ -20,7 +20,7 @@ $.extend(Drupal, {
             Drupal.l10nClient.toggle(1);
             $('#l10n-client .string-search').focus();
           } else {
-            if($('#l10n-client').is('.hidden')) {
+            if($('#l10n-client').is('.l10n-client-minimized')) {
               Drupal.l10nClient.toggle(1);
               if(!$.browser.safari) {
                 $('#l10n-client .string-search').focus();
@@ -40,7 +40,7 @@ $.extend(Drupal, {
       switch(state) {
         case 1:
           $('#l10n-client-string-select, #l10n-client-string-editor, #l10n-client .labels .label').show();
-          $('#l10n-client').height('22em').removeClass('hidden');
+          $('#l10n-client').height('22em').removeClass('l10n-client-minimized');
           $('#l10n-client .labels .toggle').text('X');
           if(!$.browser.msie) {
             $('body').css('border-bottom', '22em solid #fff');
@@ -49,7 +49,7 @@ $.extend(Drupal, {
         break;
         case 0:
           $('#l10n-client-string-select, #l10n-client-string-editor, #l10n-client .labels .label').hide();
-          $('#l10n-client').height('2em').addClass('hidden');
+          $('#l10n-client').height('2em').addClass('l10n-client-minimized');
           $('#l10n-client .labels .toggle').text(Drupal.t('Translate Text'));
           if(!$.browser.msie) {
             $('body').css('border-bottom', '0px');
@@ -123,7 +123,7 @@ Drupal.behaviors.l10nClient.attach = function (context) {

   // When l10n_client window is clicked, toggle based on current state.
   $('#l10n-client .labels .toggle').click(function() {
-    if($('#l10n-client').is('.hidden')) {
+    if($('#l10n-client').is('.l10n-client-minimized')) {
       Drupal.l10nClient.toggle(1);
     } else {
       Drupal.l10nClient.toggle(0);
diff --git a/l10n_client.module b/l10n_client.module
index 71811e2..e3287be 100644
--- a/l10n_client.module
+++ b/l10n_client.module
@@ -228,7 +228,7 @@ function l10n_client_page_alter(&$page) {
     $toggle_label = t('Translate Text');

     $output = "
-      <div id='l10n-client' class='hidden'>
+      <div id='l10n-client' class='l10n-client-minimized'>
         <div class='labels'>
           <span class='toggle'>$toggle_label</span>
           <div class='label strings'>$string_label</div>
