diff --git a/modules/user/user.module b/modules/user/user.module
index dfa05978cb..b8bbdfd13d 100644
--- a/modules/user/user.module
+++ b/modules/user/user.module
 /**
  * @file
@@ -2400,7 +2400,7 @@ function user_external_login_register($name, $module) {
  */
 function user_pass_reset_url($account) {
   $timestamp = REQUEST_TIME;
-  return url("user/reset/$account->uid/$timestamp/" . user_pass_rehash($account->pass, $timestamp, $account->login, $account->uid, $account->mail), array('absolute' => TRUE));
+  return url("user/reset/$account->uid/$timestamp/" . user_pass_rehash($account->pass, $timestamp, $account->login, $account->uid, $account->mail) . '/login', array('absolute' => TRUE));
 }
 
 /**
