diff --git a/js/user-alert.js b/js/user-alert.js
index 883b8e8..4025e6d 100644
--- a/js/user-alert.js
+++ b/js/user-alert.js
@@ -1,6 +1,9 @@
 (function ($) {
   Drupal.behaviors.user_alert_get_message = {
     attach: function(context) {
+      if ( !$('.block-user-alert').length ) {
+        return;
+      }
       $.ajax({
         type: 'GET',
         url: Drupal.settings.basePath + Drupal.settings.pathPrefix + Drupal.settings.user_alert.url_prefix + 'js/user-alert/get-message',
@@ -8,7 +11,7 @@
           $('.block-user-alert').html(response[1].data);
         }
       });
-    	
+
       $('body').delegate('div.user-alert-close a', 'click', function() {
         id = $(this).attr('rel');
         $.ajax({
@@ -20,6 +23,6 @@
           }
         });
       });
-  	}
+    }
   };
 }(jQuery));
