From d561368c178adeaef99ca155a8e9319e7ff35234 Mon Sep 17 00:00:00 2001
From: JrgenGNielsen <jgn@dbc.dk>
Date: Thu, 6 Jul 2017 16:00:38 +0200
Subject: [PATCH] fix_strict_pass_by_reference_errors.patch

---
 zabbix.module | 45 ++++++++++++++++++++++++++++++---------------
 1 file changed, 30 insertions(+), 15 deletions(-)

diff --git a/zabbix.module b/zabbix.module
index 59ca67a..dd2ae0b 100644
--- a/zabbix.module
+++ b/zabbix.module
@@ -519,7 +519,8 @@ function zabbix_module_update_status() {
  * Return the time that the admin user last logged in as a unix timestamp.
  */
 function zabbix_last_admin_login() {
-  $time = (int) reset(db_query("SELECT login FROM {users} WHERE uid = 1"));
+  $db_time = db_query("SELECT login FROM {users} WHERE uid = 1");
+  $time = (int) reset($db_time);
   return $time;
 }
 
@@ -532,7 +533,8 @@ function zabbix_watchdog_emergencies() {
     $interval = variable_get('zabbix_zabbix_watchdog_emergencies_run_interval', 86400);
     $last_run -= $interval;
   }
-  $count = (int) reset(db_query("SELECT COUNT(*) FROM {watchdog} WHERE severity = :severity and timestamp > :timestamp", array(':severity' => WATCHDOG_EMERG, ':timestamp' => $last_run)));
+  $db_count = db_query("SELECT COUNT(*) FROM {watchdog} WHERE severity = :severity and timestamp > :timestamp", array(':severity' => WATCHDOG_EMERGENCY, ':timestamp' => $last_run));
+  $count = (int) reset($db_count);
   return $count;
 }
 
@@ -545,7 +547,8 @@ function zabbix_watchdog_alerts() {
     $interval = variable_get('zabbix_zabbix_watchdog_alerts_run_interval', 86400);
     $last_run -= $interval;
   }
-  $count = (int) reset(db_query("SELECT COUNT(*) FROM {watchdog} WHERE severity = :severity and timestamp > :timestamp", array(':severity' => WATCHDOG_ALERT, ':timestamp' => $last_run)));
+  $db_count = db_query("SELECT COUNT(*) FROM {watchdog} WHERE severity = :severity and timestamp > :timestamp", array(':severity' => WATCHDOG_ALERT, ':timestamp' => $last_run));
+  $count = (int) reset($db_count);
   return $count;
 }
 
@@ -558,7 +561,8 @@ function zabbix_watchdog_critical() {
     $interval = variable_get('zabbix_zabbix_watchdog_critical_run_interval', 86400);
     $last_run -= $interval;
   }
-  $count = (int) reset(db_query("SELECT COUNT(*) FROM {watchdog} WHERE severity = :severity and timestamp > :timestamp", array(':severity' => WATCHDOG_CRITICAL, ':timestamp' => $last_run)));
+  $db_count = db_query("SELECT COUNT(*) FROM {watchdog} WHERE severity = :severity and timestamp > :timestamp", array(':severity' => WATCHDOG_CRITICAL, ':timestamp' => $last_run));
+  $count = (int) reset($db_count);
   return $count;
 }
 
@@ -571,7 +575,8 @@ function zabbix_watchdog_errors() {
     $interval = variable_get('zabbix_zabbix_watchdog_errors_run_interval', 86400);
     $last_run -= $interval;
   }
-  $count = (int) reset(db_query("SELECT COUNT(*) FROM {watchdog} WHERE severity = :severity and timestamp > :timestamp", array(':severity' => WATCHDOG_ERROR, ':timestamp' => $last_run)));
+  $db_count = db_query("SELECT COUNT(*) FROM {watchdog} WHERE severity = :severity and timestamp > :timestamp", array(':severity' => WATCHDOG_ERROR, ':timestamp' => $last_run));
+  $count = (int) reset($db_count);
   return $count;
 }
 
@@ -584,7 +589,8 @@ function zabbix_watchdog_warnings() {
     $interval = variable_get('zabbix_zabbix_watchdog_errors_run_interval', 86400);
     $last_run -= $interval;
   }
-  $count = (int) reset(db_query("SELECT COUNT(*) FROM {watchdog} WHERE severity = :severity and timestamp > :timestamp", array(':severity' => WATCHDOG_WARNING, ':timestamp' => $last_run)));
+  $db_count = db_query("SELECT COUNT(*) FROM {watchdog} WHERE severity = :severity and timestamp > :timestamp", array(':severity' => WATCHDOG_WARNING, ':timestamp' => $last_run));
+  $count = (int) reset($db_count);
   return $count;
 }
 
@@ -597,7 +603,8 @@ function zabbix_watchdog_notices() {
     $interval = variable_get('zabbix_zabbix_watchdog_errors_run_interval', 86400);
     $last_run -= $interval;
   }
-  $count = (int) reset(db_query("SELECT COUNT(*) FROM {watchdog} WHERE severity = :severity and timestamp > :timestamp", array(':severity' => WATCHDOG_NOTICE, ':timestamp' => $last_run)));
+  $db_count = db_query("SELECT COUNT(*) FROM {watchdog} WHERE severity = :severity and timestamp > :timestamp", array(':severity' => WATCHDOG_NOTICE, ':timestamp' => $last_run));
+  $count = (int) reset($db_count);
   return $count;
 }
 
@@ -610,7 +617,8 @@ function zabbix_watchdog_info() {
     $interval = variable_get('zabbix_zabbix_watchdog_info_run_interval', 86400);
     $last_run -= $interval;
   }
-  $count = (int) reset(db_query("SELECT COUNT(*) FROM {watchdog} WHERE severity = :severity and timestamp > :timestamp", array(':severity' => WATCHDOG_INFO, ':timestamp' => $last_run)));
+  $db_count = db_query("SELECT COUNT(*) FROM {watchdog} WHERE severity = :severity and timestamp > :timestamp", array(':severity' => WATCHDOG_INFO, ':timestamp' => $last_run));
+  $count = (int) reset($db_count);
   return $count;
 }
 
@@ -623,7 +631,8 @@ function zabbix_watchdog_debug() {
     $interval = variable_get('zabbix_zabbix_watchdog_debug_run_interval', 86400);
     $last_run -= $interval;
   }
-  $count = (int) reset(db_query("SELECT COUNT(*) FROM {watchdog} WHERE severity = :severity and timestamp > :timestamp", array(':severity' => WATCHDOG_DEBUG, ':timestamp' => $last_run)));
+  $db_count = db_query("SELECT COUNT(*) FROM {watchdog} WHERE severity = :severity and timestamp > :timestamp", array(':severity' => WATCHDOG_DEBUG, ':timestamp' => $last_run));
+  $count = (int) reset($db_count);
   return $count;
 }
 
@@ -636,7 +645,8 @@ function zabbix_php_watchdog_entries() {
     $interval = variable_get('zabbix_zabbix_watchdog_errors_run_interval', 86400);
     $last_run -= $interval;
   }
-  $count = (int) reset(db_query("SELECT COUNT(*) FROM {watchdog} WHERE type = 'php' and timestamp > :type", array(':type' => $last_run)));
+  $db_count = db_query("SELECT COUNT(*) FROM {watchdog} WHERE type = 'php' and timestamp > :type", array(':type' => $last_run));
+  $count = (int) reset($db_count);
   return $count;
 }
 
@@ -644,7 +654,8 @@ function zabbix_php_watchdog_entries() {
  * Count the total number of users on the site
  */
 function zabbix_count_all_users() {
-  $count = (int) reset(db_query("SELECT COUNT(*) FROM {users} WHERE uid != 0 AND uid IS NOT NULL"));
+  $db_count = db_query("SELECT COUNT(*) FROM {users} WHERE uid != 0 AND uid IS NOT NULL");
+  $count = (int) reset($db_count);
   return $count;
 }
 
@@ -652,7 +663,8 @@ function zabbix_count_all_users() {
  * Count the number of active users on the site
  */
 function zabbix_count_active_users() {
-  $count = (int) reset(db_query("SELECT COUNT(*) FROM {users} WHERE status = 1 AND uid != 0 AND uid IS NOT NULL"));
+  $db_count = db_query("SELECT COUNT(*) FROM {users} WHERE status = 1 AND uid != 0 AND uid IS NOT NULL");
+  $count = (int) reset($db_count);
   return $count;
 }
 
@@ -684,7 +696,8 @@ function zabbix_list_themes() {
  * Count the number of published nodes on the site.
  */
 function zabbix_count_published_nodes() {
-  $count = (int) reset(db_query("SELECT COUNT(DISTINCT nid) FROM {node} WHERE status = 1"));
+  $db_count = db_query("SELECT COUNT(DISTINCT nid) FROM {node} WHERE status = 1");
+  $count = (int) reset($db_count);
   return $count;
 }
 
@@ -692,7 +705,8 @@ function zabbix_count_published_nodes() {
  * Count the total number of node on the site.
  */
 function zabbix_count_all_nodes() {
-  $count = (int) reset(db_query("SELECT COUNT(DISTINCT nid) FROM {node}"));
+  $db_count = db_query("SELECT COUNT(DISTINCT nid) FROM {node}");
+  $count = (int) reset($db_count);
   return $count;
 }
 
@@ -715,7 +729,8 @@ function zabbix_count_sessions_anon() {
  */
 function zabbix_count_sessions_auth() {
   $interval = time() - 900;
-  $count = (int) reset(db_query("SELECT COUNT(sid) FROM {sessions} WHERE timestamp >= :timestamp", array(':timestamp' => $interval)));
+  $db_count = db_query("SELECT COUNT(sid) FROM {sessions} WHERE timestamp >= :timestamp", array(':timestamp' => $interval));
+  $count = (int) reset($db_count);
   return $count ? $count : 0;
 }
 
-- 
1.9.1

