diff --git a/l10n_community/l10n_community.admin.inc b/l10n_community/l10n_community.admin.inc
index 7c1bd7d..676f41a 100644
--- a/l10n_community/l10n_community.admin.inc
+++ b/l10n_community/l10n_community.admin.inc
@@ -34,6 +34,12 @@ function l10n_community_settings_form($form, &$form_state) {
     '#default_value' => variable_get('l10n_community_import_user', ''),
     '#autocomplete_path' => 'user/autocomplete',
   );
+  $form['l10n_community_override_suggestions'] = array(
+    '#title' => t('Override the default suggestions feature'),
+    '#description' => t('If checked, will incomming translations be accepted right away'),
+    '#type' => 'checkbox',
+    '#default_value' => variable_get('l10n_community_override_suggestions', FALSE),
+  );
   $form['l10n_community_highlighted_project'] = array(
     '#title' => t('Highlight project on the welcome page'),
     '#description' => t('Pick a project which will be highlighted on the welcome page, so progress of all translation teams can be seen instantly.'),
diff --git a/l10n_community/l10n_community.module b/l10n_community/l10n_community.module
index 948e42a..b26583d 100644
--- a/l10n_community/l10n_community.module
+++ b/l10n_community/l10n_community.module
@@ -972,6 +972,7 @@ function l10n_community_ajax_releases($form, $form_state) {
  *   User ID to use to keep history of.
  * @param $medium
  *   Medium type constant L10N_SERVER_MEDIUM_*
+ * @return bool|\DatabaseStatementInterface|int|null
  */
 function l10n_community_add_suggestion($sid, $translation, $langcode, $uid_attribution, $uid_user, $medium) {
 
@@ -1042,7 +1043,10 @@ function l10n_community_add_suggestion($sid, $translation, $langcode, $uid_attri
 
   // Mark the existing or mock translation as having suggestions.
   l10n_community_update_string_status($langcode, $sid);
-
+  // If we're accepting translations right away approve it.
+  if(variable_get('l10n_community_override_suggestions', FALSE) == TRUE){
+    l10n_community_approve_string($langcode, $sid, $tid);
+  }
   return $tid;
 }
 
diff --git a/l10n_server/l10n_server.install b/l10n_server/l10n_server.install
index ddf19eb..5583769 100644
--- a/l10n_server/l10n_server.install
+++ b/l10n_server/l10n_server.install
@@ -70,6 +70,12 @@ function l10n_server_schema() {
         'not null' => TRUE,
         'default' => 0,
       ),
+      'accept_suggestions' => array(
+        'description' => 'Whether the project will surpress the suggestion feature and accept suggestions right away = 1, 0 = new translations will be considered as suggestions',
+        'type' => 'int',
+        'not null' => TRUE,
+        'default' => 0,
+      ),
     ),
     'primary key' => array('pid'),
     'indexes' => array(
@@ -458,3 +464,15 @@ function l10n_server_update_7000() {
     'default' => 0,
   ));
 }
+
+/**
+ * Add field to hold suggestion value for projects
+ */
+function l10n_server_update_7001() {
+  db_add_field('l10n_server_project', 'accept_suggestions', array(
+    'description' => 'Whether the project will surpress the suggestion feature and accept suggestions right away = 1, 0 = new translations will be considered as suggestions',
+    'type' => 'int',
+    'not null' => TRUE,
+    'default' => 0,
+  ));
+}
diff --git a/l10n_server/l10n_server.projects.inc b/l10n_server/l10n_server.projects.inc
index 033b1a0..1ccadae 100644
--- a/l10n_server/l10n_server.projects.inc
+++ b/l10n_server/l10n_server.projects.inc
@@ -315,6 +315,7 @@ function l10n_server_admin_projects_cleanup_form_submit($form, &$form_state) {
  * Project editing and submission form.
  */
 function l10n_server_admin_projects_edit_form($form, &$form_state, $project_uri = NULL) {
+
   if (!isset($project_uri)) {
     $connector_list = l10n_server_get_add_enabled_project_connectors();
     // Provide sensible default for project if adding new one.
@@ -373,6 +374,12 @@ function l10n_server_admin_projects_edit_form($form, &$form_state, $project_uri
     '#description' => t('Weight of the project. Lighter projects float up when ordering by weight.'),
     '#default_value' => $project->weight,
   );
+  $form['accept_suggestions'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Override suggestions'),
+    '#description' => t('NOT YET FUNCTIONAL! Should this project accepts submissions right away? If checked incoming translations will be accepted right away'),
+    '#default_value' => 0,
+  );
   $form['submit'] = array(
     '#type' => 'submit',
     '#value' => t('Save project'),
@@ -402,13 +409,12 @@ function l10n_server_admin_projects_edit_form_validate($form, &$form_state) {
  */
 function l10n_server_admin_projects_edit_form_submit($form, &$form_state) {
   $new_project = $form_state['values']['previous_project'];
-  foreach (array('uri', 'title', 'home_link', 'connector_module', 'weight') as $property) {
+  foreach (array('uri', 'title', 'home_link', 'connector_module', 'weight', 'accept_suggestions') as $property) {
     if (isset($form_state['values'][$property])) {
       $new_project->$property = $form_state['values'][$property];
     }
   }
   $new_project->weight = (int) $new_project->weight;
-
   drupal_write_record('l10n_server_project', $new_project, !empty($new_project->pid) ? 'pid' : array());
   drupal_set_message(t('Project data saved.'));
   $form_state['redirect'] = 'admin/l10n_server/projects';
