diff --git a/tests/CacheEntryTest.php b/tests/CacheEntryTest.php
index 89ced2c..b9a1ce4 100644
--- a/tests/CacheEntryTest.php
+++ b/tests/CacheEntryTest.php
@@ -3,13 +3,14 @@
 namespace Kevinrob\GuzzleCache\Tests;

 use Kevinrob\GuzzleCache\CacheEntry;
+use PHPUnit\Framework\TestCase;
 use Psr\Http\Message\RequestInterface;
 use Psr\Http\Message\ResponseInterface;

 /**
  * @group time-sensitive
  */
-class CacheEntryTest extends \PHPUnit_Framework_TestCase
+class CacheEntryTest extends TestCase
 {
     /**
      * @var RequestInterface
