diff --git a/l10n_client.module b/l10n_client.module
index acb321b..eb6f423 100644
--- a/l10n_client.module
+++ b/l10n_client.module
@@ -476,7 +476,7 @@ function l10n_client_save_string() {
         // Submit to remote server if enabled.
         if (variable_get('l10n_client_use_server', FALSE) && user_access('submit translations to localization server') && ($_POST['textgroup'] == 'default')) {
           if (!empty($user->data['l10n_client_key'])) {
-            $remote_result = l10n_client_submit_translation($language->language, $_POST['source'], $_POST['target'], $user->data['l10n_client_key'], l10n_client_user_token($user));
+            $remote_result = l10n_client_submit_translation($language->language, $_POST['source'], $_POST['target'], $user->data['l10n_client_key'], l10n_client_user_token($user), $_POST['context']);
             $message .= theme('l10n_client_message', array('message' => $remote_result[1], 'level' => $remote_result[0] ? WATCHDOG_INFO : WATCHDOG_ERROR));
           }
           else {
@@ -538,6 +538,12 @@ function l10n_client_settings_form() {
     '#description' => t('Each translation submission will also be submitted to this server. We suggest you enter <a href="@localize">http://localize.drupal.org</a> to share with the greater Drupal community. Make sure you set up an API-key in the user account settings for each user that will participate in the translations.', array('@localize' => 'http://localize.drupal.org')),
     '#default_value' => variable_get('l10n_client_server', 'http://localize.drupal.org'),
   );
+  $form['l10n_client_localisation_project'] = array(
+    '#title' => t('Project'),
+    '#type' => 'textfield',
+    '#description' => t('What project on the localisation server should this connection refer to? The string you enter in this field should be equal the value entered in the URI field found on the project setup page on the localisation server'),
+    '#default_value' => variable_get('l10n_client_localisation_project', NULL),
+  );
   return system_settings_form($form);
 }
 
@@ -635,10 +641,11 @@ function l10n_client_user_token($account = NULL) {
 /**
  * Submit translation to the server.
  */
-function l10n_client_submit_translation($langcode, $source, $translation, $user_key, $user_token) {
+function l10n_client_submit_translation($langcode, $source, $translation, $user_key, $user_token, $context) {
   $server_uid = current(explode(':', $user_key));
   $signature = md5($user_key . $langcode . $source . $translation . $user_token);
   $server_url = variable_get('l10n_client_server', 'http://localize.drupal.org');
+  $project = variable_get('l10n_client_localisation_project', NULL);
 
   $response = xmlrpc(
     $server_url .'/xmlrpc.php',
@@ -648,7 +655,9 @@ function l10n_client_submit_translation($langcode, $source, $translation, $user_
       $translation,
       (int)$server_uid,
       $user_token,
-      $signature
+      $signature,
+      $context,
+      $project
     ))
   );
 
